src/clients/Device.js

```diff
@@ -327,14 +318,18 @@ class Device extends EventEmitter {
       });*/
 
       this._sendHello();
-
       this._connectionStartTime = new Date();
 
-      // We don't need to await this as other functions will await the response
```

Comment:
yes we do. function that await for this promise is `get attributes from database and setting state to READY`, if we don't await description, we can't guarantee strict status order. It should be strongly INTIAL -> GOT_HELLO -> GOT_DESCRIPTION -> READY.